<launch>
  <node pkg="voice_common_pkg" name="stt_server" type="stt_server.py" output="screen" />
  <node pkg="gcp_texttospeech" name="tts_srvserver" type="tts_srvserver.py" output="screen" />
  <node pkg="voice_common_pkg" name="listen_command" type="listen_command.py" output="screen" />
  <node pkg="sp_ggi" name="ggi_learning" type="ggi_learning.py" output="screen" />
  <node pkg="voice_common_pkg" name="yes_no" type="yes_no.py" output="screen" />
  <node pkg="sp_ggi" name="test_phase" type="test_phase.py" output="screen" />
</launch>
